lib/unittest_server.py

```diff
@@ -17,6 +17,7 @@ def __init__(self, *args, **kwargs):
     def execute(self, server):
         server.current_test = self
         execs = server.prepare_args()
+        execs[0] = os.path.abspath(execs[0])
```

Comment:
I looked over `TarantoolServer` and `AppServer` and construct an absolute path in `prepare_args()`. So I[ll follow this approach here too:

```diff
diff --git a/lib/unittest_server.py b/lib/unittest_server.py
index 9d57558..5770941 100644
--- a/lib/unittest_server.py
+++ b/lib/unittest_server.py
@@ -50,7 +50,7 @@ class UnittestServer(Server):
     def prepare_args(self, args=[]):
         executable_path = os.path.join(self.builddir, "test",
                                        self.current_test.name)
-        return [executable_path] + args
+        return [os.path.abspath(executable_path)] + args
 
     def deploy(self, vardir=None, silent=True, wait=True):
         self.vardir = vardir
```

However you way is okay too. The decision is up to you.

Reply:
I like you fix :) so I updated a patch.